packages/dynamodb/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@next-auth/example-adapter",
```

Comment:
Great job! I stopped work on the implementation, I'll test soon yours. Just update this little left over in order to the package have a proper name on npm.

Reply:
Thanks for the heads up ! I've updated the name :) Let me know if you catch any bug while testing